sdk/state/ingest_test.go

```diff
@@ -1033,35 +1033,48 @@ func TestChannel_IngestTx_seqNumCantGoBackwards(t *testing.T) {
 		require.NoError(t, err)
 		assert.Equal(t, StateOpen, cs)
 	}
```

Comment:
had to refactor this test a bit testing the state doesn't move backwards because of this PR change